modules/openapi-generator/src/main/resources/kotlin-multiplatform-client/model.mustache

```diff
@@ -1,18 +1,20 @@
 {{>licenseInfo}}
-@file:UseSerializers(InstantSerializer::class, LocalDateTimeSerializer::class, LocalDateSerializer::class)
+{{#dateLibraryKotlinx}}@file:UseSerializers(InstantSerializer::class, LocalDateTimeSerializer::class, LocalDateSerializer::class){{/dateLibraryKotlinx}}
 @file:Suppress("UnusedImport", "RemoveRedundantQualifierName")
 
 package {{modelPackage}}
 
-import kotlinx.serialization.UseSerializers
-import {{packageName}}.infrastructure.LocalDateTimeSerializer
-import {{packageName}}.infrastructure.InstantSerializer
-import {{packageName}}.infrastructure.LocalDateSerializer
+{{#dateLibraryKotlinx}}
+    import kotlinx.serialization.UseSerializers
```

Comment:
You have to look out for the indentation, best not use a plugin for mustache. If it's indented here it also will be in the final file.

Reply:
Done